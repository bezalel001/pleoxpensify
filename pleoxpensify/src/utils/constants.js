export const LIMIT = 25;
export const BASE_URL = 'http://localhost:3000/expenses';
