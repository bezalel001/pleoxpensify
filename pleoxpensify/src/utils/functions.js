import moment from 'moment';

export const formatAmount = ({ currency, value }) => {
  return new Intl.NumberFormat('en-UK', {
    style: 'currency',
    currency
  }).format(parseInt(value, 10));
};

export const formatDate = date => moment(date).format('MMM Do, YYYY');
