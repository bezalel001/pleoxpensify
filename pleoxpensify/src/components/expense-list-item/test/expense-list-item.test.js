import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../index';
import { expenses } from '../../../utils/fixtures';

describe('ExpenseListItem', () => {
  it('should render ExpenseListItem correctly', () => {
    const wrapper = shallow(<ExpenseListItem expense={expenses[0]} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should have an expense-list-item class', () => {
    const wrapper = shallow(<ExpenseListItem expense={expenses[0]} />);
    expect(wrapper.find('.expense-list-item').length > 0).toBe(true);
  });
});
