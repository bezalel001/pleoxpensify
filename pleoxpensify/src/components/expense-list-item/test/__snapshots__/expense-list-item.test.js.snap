// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpenseListItem should render ExpenseListItem Correctly 1`] = `ShallowWrapper {}`;

exports[`ExpenseListItem should render ExpenseListItem correctly 1`] = `ShallowWrapper {}`;
